electron/update.js

```diff
@@ -65,8 +65,8 @@ export default (icon, notify) => {
       buttons: ['Restart', 'Later'],
       title: 'React Native Debugger',
       icon,
-      message: 'The new version has been downloaded. ' +
-        'Please restart the application to apply the updates.',
+      message: 'The newer version has been downloaded. ' +
```

Comment:
This could also be “The update…”, but I thought that maybe using the word “update” twice in this dialog was a bit much. What do you think?

Reply:
> This could also be “The update…”, but I thought that maybe using the word “update” twice in this dialog was a bit much.

Agreed, I'm 👍 for keeping it.

Reply:
Could also be "The update has been downloaded, please restart the application to apply it." - As the update is the subject of the statement, it is implied that it continues to be, so you don't have to repeat yourself and you can make it a bit shorter and flow better by making it once sentence with a comma :slightly_smiling_face:

I'd even say you could omit the "to apply the update"; "The update has been downloaded, please restart the application." - imo it's obvious enough why you are restarting, again because the update is the subject it's easy enough to put two and two together :wink: